virtual_agent_position: [0.0,0.0,0.0]
crazyflie1: [0.25,0.0,0.5]
crazyflie2: [-0.25,0.25,0.5]
crazyflie3: [-0.25,-0.25,0.5]
translation_x: 0.0
translation_y: 0.0
translation_z: 0.0
scaling: 1
rotation: 0.0
